diff --git a/node_modules/@isaul32/ckeditor5-math/src/utils.js b/node_modules/@isaul32/ckeditor5-math/src/utils.js
index f2ddcfc..2017bc6 100644
--- a/node_modules/@isaul32/ckeditor5-math/src/utils.js
+++ b/node_modules/@isaul32/ckeditor5-math/src/utils.js
@@ -13,7 +13,7 @@ export function getSelectedMathModelWidget(selection) {
 export function isMathJaxVersion3(MathJax) {
     return (MathJax != null && typeof MathJax == 'object' && 'version' in MathJax && typeof MathJax.version == 'string' &&
         MathJax.version.split('.').length === 3 &&
-        MathJax.version.split('.')[0] === '3');
+        ['3', '4'].includes(MathJax.version.split('.')[0]));
 }
 // Simple MathJax 2 version check
 export function isMathJaxVersion2(MathJax) {
